docs/getting-started/12-bind-redis-service-to-function.md

```diff
@@ -0,0 +1,212 @@
+  </summary>
+
+1. Go to **Development** > **Functions** in the left navigation panel and select `orders-function`.
+
+2. Switch to the **Configuration** tab and select **Create Service Binding** in the **Service Bindings** section.
+
+3. Select `redis-service` in the **Service Instance** drop-down list, add `REDIS_` as **Prefix for injected variables**, and make sure **Create new Secret** is selected.
+
+   > **NOTE:** The **Prefix for injected variables** field is optional. It adds a prefix to all environment variables injected in a Secret to the Function when creating a ServiceBinding. In our example, the prefix is set to `REDIS_`, so all environmental variables will follow the `REDIS_{ENVIRONMENT_VARIABLE}` naming pattern.
+
+   > **TIP:** It is considered good practice to use prefixes for environment variables. In some cases, a Function must use several instances of a given ServiceClass. Prefixes allow you to distinguish between instances and make sure t
+
+4. Select **Create** to confirm the changes.
+
+A message will appear on the screen confirming that the ServiceBinding was created, and you will see it in the **Service Bindings** section in your Function along with environment variable names.
+
+If you switch to the **Code** tab and scroll down to the **Environment Variables** section, you should see `REDIS_PORT`, `REDIS_HOST` and `REDIS_REDIS_PASSWORD` items with the `Service Binding` type. It indicates that the environment variable was injected to the Function by the ServiceBinding.
+
+    </details>
+</div>
+
+### Call and test the Function
+
+Follow these steps:
+
+> **CAUTION:** If you have a Minikube cluster, you must first add the IP address of the exposed Service to the `hosts` file on your machine:
+>
+>  ```bash
+>  echo "$(minikube ip) orders-service.kyma.local" | sudo tee -a /etc/hosts
+>  ```
+
+1. Retrieve the domain of the exposed Function and save it to the environment variable:
+
+   ```bash
+   export FUNCTION_DOMAIN=$(kubectl get virtualservices -l apirule.gateway.kyma-project.io/v1alpha1=orders-function.orders-service -n orders-service -o=jsonpath='{.items[*].spec.hosts[0]}')
+   ```
+
+2. Run this command in the terminal to call the Function:
+
+   ```bash
+   curl -ik "https://$FUNCTION_DOMAIN"
+   ```
+
+   The system returns a response similar to this one:
+
+   ```bash
+   HTTP/2 200
+   access-control-allow-origin: *
+   content-length: 187
+   content-type: application/json; charset=utf-8
+   date: Mon, 13 Jul 2020 13:05:33 GMT
+   etag: W/"bb-Iuoc/aX9UjdqADJAZPNrwPdoraI"
+   server: istio-envoy
+   x-envoy-upstream-service-time: 25
+   x-powered-by: Express
+
+   [{"orderCode":"762727210","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}, {"orderCode":"123456789","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}]
+   ```
+
+   > **NOTE:** The listed orders are those created in the previous guides.
+
+3. Send a `POST` request to the Function with a sample order details:
```

Comment:
```suggestion
3. Send a `POST` request to the Function with sample order details:
```
See [this comment](https://github.com/kyma-project/kyma/pull/9296#discussion_r480108879).